test/load_brocfile_test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const loadBrocfile = require('../lib/load_brocfile');
+const chai = require('chai');
+
+const projectPath = 'test/fixtures/project';
+const brocfileFixture = require('../' + projectPath + '/Brocfile.js');
+
+describe('loadBrocfile', function() {
+  let oldCwd = null;
+
+  beforeEach(function() {
+    oldCwd = process.cwd();
+  });
+
+  afterEach(function() {
+    process.chdir(oldCwd);
+  });
+
+  context('with Brocfile.js', function() {
+    beforeEach(function() {
+      process.chdir(projectPath);
+    });
+
+    it('return tree definition', function() {
+      chai.expect(loadBrocfile()).to.equal(brocfileFixture);
+    });
+  });
+
+  context('with Brocfile.js, called in subfolder', function() {
+    beforeEach(function() {
+      process.chdir(projectPath + '/subdir');
+    });
+
+    afterEach(function() {
+      process.chdir(oldCwd);
+    });
+
+    it('return tree definition', function() {
+      chai.expect(loadBrocfile()).to.equal(brocfileFixture);
+    });
+  });
+
+  context('with path', function() {
+    it('return tree definition', function() {
+      chai.expect(loadBrocfile(projectPath + '/Brocfile.js')).to.equal(brocfileFixture);
```

Comment:
can we also test the error case here? Specifically that if an explicit path is given, but that path doesn't contain a file that we hit the helpful error?

Reply:
make sense :+1:

Reply:
thx

Reply:
done